lib/ox/element.rb

```diff
@@ -150,7 +150,18 @@ def locate(path)
     # _raise_ [NoMethodError] if no match is found
     def method_missing(id, *args, &block)
       has_some = false
-      ids = id.to_s
+      ids = id.to_s.sub('_', ':') # allows easy API with namespaces
```

Comment:
I don't know how to call a method that has a `:` in the name. The only non-alpha-num chars that are allowed in methods names to my knowledge are `_`, `?` and `!`, which is why I opted for `_` here. I have no strong preference for what character is used -- in fact, if `:` can be used, that would be better. I just don't know how.